sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 5.0.0 (2020-07-27)
+
+- This release is to correct the package version to be 5.0.0 instead of 1.0.0.
```

Comment:
We should include a note on why 5.0.0 is deemed to be the "correct" version here. Changelog should either contain enough details on the why or link to a PR that contains those details. All this in an attempt to be more transparent about why we do what we do

Reply:
Also, do we expect this release to include the `text` property related changes we are doing in another PR?
Or is this more of an out of cycle release to get the version number right

Reply:
done. It is an out of cycle release to get the version number right but I will try to add the fix to the `text` property included as well.

Reply:
we decided to add the fix in the next release instead.